ESPixelStick/ESPixelStick.ino

```diff
@@ -76,25 +59,37 @@ static void _u0_putc(char c){
 //  Globals
 //
 /////////////////////////////////////////////////////////
+// OLED
+#include <SPI.h>
+#include <Wire.h>
+#include <Adafruit_GFX.h>
+#include <Adafruit_SSD1306.h>
+
+#define SCREEN_WIDTH 128    // OLED display width, in pixels
+#define SCREEN_HEIGHT 32    // OLED display height, in pixels
+#define OLED_RESET -1       // Reset pin # (or -1 if sharing Arduino reset pin)
+#define SCREEN_ADDRESS 0x3C ///< See datasheet for Address; 0x3D for 128x64, 0x3C for 128x32
+Adafruit_SSD1306 display(SCREEN_WIDTH, SCREEN_HEIGHT, &Wire, OLED_RESET);
 
 #ifdef ESPS_VERSION
 const String VERSION = STRING(ESPS_VERSION);
 #else
-const String VERSION = "4.x-dev";
+const String VERSION = "4.x-jmt";
```

Comment:
Please restore this line. This is done using the platform name from platformio.ini